TESTING.md

```diff
@@ -0,0 +1,10 @@
+## Testing
+
+`canvas-kit` has 3 levels of testing: unit, functional and visual. Each area has special use-cases even though what they test might overlap. Here's a simplified explaination of what each level is meant to test:
+
+- *unit*: This level is meant to test the expected outputs of developer APIs (ex: When a developer passes in X prop, then expect a specific output
+- *functional*: This level is browser-based testing and is where we specify UI behavior and accessibility requirements
```

Comment:
Also where we test combinations of components (i.e. `FormField` + `Checkbox`)

Reply:
Right. I've heard "Patterns" and "Use Cases". I'm not sure what to call them. Basically they are stories that should use-cases where a component to encapsulate the use-case would be too complicated and rigid or contain too much customization and so we leave them to composition patterns.